recipes/crisprme/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: crisprme
+  version: {{ version }}
+
+source:
+  url: https://github.com/samuelecancellieri/CRISPRme/archive/v{{ version }}.tar.gz
+  sha256: 427c755b0c489ded70bb7ada4b347302eab31aa51439c36e8ccf6375db3d4884
+  
+build:
+  number: 0
+  skip: True  # [osx or py<36]

```

Comment:
This appears to be pure python/shell script package, so why is OSX not supported?

Reply:
crispritz package i use as core, is not supported by OSX, so installing this package is completely useless in OSX due to this incompatibility

Reply:
OK, you can add `noarch: generic` then and remove the `skip:` line. This still won't be installable on OSX until crispritz supports OSX, so there's no issue there.

Reply:
done